package nhat.fpt.Mock1.exception;

public class ExceptionResponse extends RuntimeException{
}
